third_party/silabs/silabs_board.gni

```diff
@@ -130,8 +130,8 @@ if (silabs_board == "BRD4304A") {
 } else if (silabs_board == "BRD4338A") {
   silabs_family = "SiWx917-common"
   silabs_mcu = "SiWG917M111MGTBA"
-  disable_lcd = false
-  show_qr_code = true
+  disable_lcd = true
```

Comment:
This is to be removed, just disabled the lcd and by default since it creates conflicts with the sleep